shared/js/otSdkDriver.js

```diff
@@ -560,8 +563,6 @@ loop.OTSdkDriver = (function() {
      * https://tokbox.com/opentok/libraries/client/js/reference/StreamEvent.html
      */
     _onRemoteStreamCreated: function(event) {
-      this._notifyMetricsEvent("Session.streamCreated");
```

Comment:
Just making sure that we want to have `Session.streamCreated` be non-screen-only, so comparing this to previous values will be "wrong." (As opposed to keeping this original event's meaning and adding a separate `Session.screen.streamCreated` for just screen.)